README.adoc

```diff
@@ -321,7 +321,10 @@ Panic! Then completely regenerate the site via:
 
 == License
 
-The content of this repository is released under the link:http://www.apache.org/licenses/LICENSE-2.0.txt[ASL 2.0].
+The articles, blog posts and other content of this repository is released under the link:http://creativecommons.org/licenses/by-sa/3.0/[Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0)].
+
+Sample code available on this content is released under the link:http://www.apache.org/licenses/LICENSE-2.0.html[Apache Software License 2.0].
```

Comment:
No I'm referring to the content mentioned in the previous paragraph. This README is tricky, as it refers to both the generated and published site, and to the source code repository. Not sure how to make the clear distinction?